src/modules/overlay/getters.js

```diff
@@ -1,10 +1,22 @@
+/**
+ * Search overlays by a search function
+ * @param {Object[]} overlays - The list of overlays
+ * @param {Function} fn - The search function
+ * @return {boolean} The resulting flag
+ */
+function findInOverlays(overlays, fn) {
+    const values = Object.keys(overlays).map(key => overlays[key]);
+    return !!values.find(fn);
+}
+
 /**
  * Check whether there are any open overlays
  * @param {Object} state - The vuex state object
  * @return {boolean} The resulting flag
  */
 export const hasOpenOverlays = state => {
-    return Object.values(state.overlays).some(({ isOpen }) => isOpen);
+    const fn = ({ isOpen }) => isOpen;
```

Comment:
why did you revert this?

Reply:
`Object.values` needs a polyfill in IE. If it doesn't have great benefits or is absolutely necessary, I don't see why this is necessary.